packer-plugin-sdk/packer/ui.go

```diff
@@ -0,0 +1,152 @@
+package packer
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/signal"
+	"strings"
+	"sync"
+	"syscall"
+
+	getter "github.com/hashicorp/go-getter"
```

Comment:
Packer uses go-getter@v2
```suggestion
	getter "github.com/hashicorp/go-getter/v2"
```

Reply:
Ah, good catch there !

Reply:
seriously, good catch Moss! Thanks!